crates/containerd-shim-wasmtime/src/instance.rs

```diff
@@ -302,9 +306,32 @@ where
         log::debug!("loading wasm component");
 
         wasmtime_wasi::runtime::in_tokio(async move {
-            tokio::select! {
-                status = self.execute_component_async(ctx, component, func, stdio) => { status }
-                sig = wait_for_signal() => { sig }
+            let mut done = false;
```

Comment:
question: what is this used for?

Reply:
I think we can come up with a better name for this, like `force_shutdown`. This variable tracks if we already initiated a shutdown and upon receiving a second `ctrl_c` signal, we cancel waiting for the tasks and exit immediately.

Reply:
agreed that `force_shutdown` would be a better name